flink-connector-oracle-cdc/src/main/java/com/ververica/cdc/connectors/oracle/OracleSource.java

```diff
@@ -0,0 +1,177 @@
+         * connector will monitor every non-system table in each monitored database.
+         */
+        public Builder<T> tableList(String... tableList) {
+            this.tableList = tableList;
+            return this;
+        }
+
+        /**
+         * An optional list of regular expressions that match schema names to be monitored; any
+         * schema name not included in the whitelist will be excluded from monitoring. By default
+         * all non-system schemas will be monitored.
+         */
+        public Builder<T> schemaList(String... schemaList) {
+            this.schemaList = schemaList;
+            return this;
+        }
+
+        /** Name of the Oracle database to use when connecting to the Oracle database server. */
+        public Builder<T> username(String username) {
+            this.username = username;
+            return this;
+        }
+
+        /** Password to use when connecting to the Oracle database server. */
+        public Builder<T> password(String password) {
+            this.password = password;
+            return this;
+        }
+
+        /** The Debezium Oracle connector properties. For example, "snapshot.mode". */
+        public Builder<T> debeziumProperties(Properties properties) {
+            this.dbzProperties = properties;
+            return this;
+        }
+
+        /**
+         * The deserializer used to convert from consumed {@link
+         * org.apache.kafka.connect.source.SourceRecord}.
+         */
+        public Builder<T> deserializer(DebeziumDeserializationSchema<T> deserializer) {
+            this.deserializer = deserializer;
+            return this;
+        }
+
+        /** Specifies the startup options. */
+        public Builder<T> startupOptions(StartupOptions startupOptions) {
+            this.startupOptions = startupOptions;
+            return this;
+        }
+
+        public DebeziumSourceFunction<T> build() {
+            Properties props = new Properties();
+            props.setProperty("connector.class", OracleConnector.class.getCanonicalName());
+            // Logical name that identifies and provides a namespace for the particular Oracle
+            // database server being
+            // monitored. The logical name should be unique across all other connectors, since it is
+            // used as a prefix
+            // for all Kafka topic names emanating from this connector. Only alphanumeric characters
+            // and
+            // underscores should be used.
+            props.setProperty("database.server.name", DATABASE_SERVER_NAME);
+            props.setProperty("database.hostname", checkNotNull(hostname));
+            props.setProperty("database.user", checkNotNull(username));
+            props.setProperty("database.password", checkNotNull(password));
+            props.setProperty("database.port", String.valueOf(port));
+            props.setProperty("database.history.skip.unparseable.ddl", String.valueOf(true));
```

Comment:
The property "database.history.skip.unparseable.ddl" means 
> A Boolean value that specifies whether the connector should ignore malformed or unknown database statements or stop processing so a human can fix the issue. The safe default is false. **Skipping should be used only with care as it can lead to data loss or mangling when the binlog is being processed.**

I think maybe it's better to make the property to be configurable instead of a fix value "true";

Reply:
I think using `true` is okay for stability consideration just like what we did in` mysql-cdc`